components/NavTabs.tsx

```diff
@@ -17,7 +17,9 @@ const NavTabs: FC<Props> = ({ tabs, linkProps }) => {
   const router = useRouter();
   return (
     <>
-      <nav className="-mb-px flex rtl:space-x-reverse space-x-2 sm:rtl:space-x-reverse space-x-5" aria-label="Tabs">
+      <nav
+        className="-mb-px flex rtl:space-x-reverse space-x-2 sm:rtl:space-x-reverse space-x-5"
```

Comment:
`space-x-2` and `space-x-5` together don't make sense. Are you sure this is intended?

Reply:
Odd, this was a merge from main. Let me check.

Reply:
@alishaz-polymath It's the same as main, looks like some minor formatting difference - https://github.com/calcom/cal.com/blob/main/components/NavTabs.tsx#L20